llvm/test/CodeGen/X86/pr108731.ll

```diff
@@ -0,0 +1,40 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -mtriple=x86_64-gnu-unknown -mcpu=znver3 | FileCheck %s
+
+define dso_local i64 @foo(i64 %0, i64 %1, i64 %2, i64 %3) local_unnamed_addr {
```

Comment:
Test cleanup (same for fooVec):
remove dso_local  / local_unnamed_addr noise
use named variables not numbers
rename foo / fooVec with descriptive names

Reply:
Hopefully the names I chose are ok 👍